packages/app/src/containers/TextStyleInspector.tsx

```diff
@@ -0,0 +1,220 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { Divider } from 'noya-designsystem';
+import { Selectors } from 'noya-state';
+import { useCallback, memo, useMemo } from 'react';
+import useShallowArray from '../hooks/useShallowArray';
+import TextOptionsRow from '../components/inspector/TextOptionsRow';
+import TextAlignmentRow from '../components/inspector/TextAlignmentRow';
+import TextStyleRow from '../components/inspector/TextStyleRow';
+import {
+  useSelector,
+  useApplicationState,
+} from '../contexts/ApplicationStateContext';
+
+/**
+ * TODO: Handle mixed styles
+ * TODO: Maybe some of this style to a selector
+ */
+
+export default memo(function TextStyleInspector() {
+  const [, dispatch] = useApplicationState();
+
+  //Todo: Handle multiple texts
+  const selectedTextStyles = useShallowArray(
+    useSelector(Selectors.getSelectedTextStyles),
+  );
+
+  const seletedText = useShallowArray(useSelector(Selectors.getSelectedText));
+
+  const fontColor = useMemo(
+    () =>
+      selectedTextStyles.map(
+        (style) =>
+          style?.textStyle?.encodedAttributes.MSAttributedStringColorAttribute,
+      ),
+    [selectedTextStyles],
+  );
+
+  const fontFamily = useMemo(
+    () =>
+      selectedTextStyles.map(
+        (style) =>
+          style?.textStyle?.encodedAttributes.MSAttributedStringFontAttribute
+            .attributes.name,
+      ),
+    [selectedTextStyles],
+  );
+
+  const fontSize = useMemo(
+    () =>
+      selectedTextStyles.map(
+        (style) =>
+          style?.textStyle?.encodedAttributes.MSAttributedStringFontAttribute
+            .attributes.size || 24,
+      ),
+    [selectedTextStyles],
+  );
+
+  const firstColor = useMemo(
+    () =>
+      fontColor[0] ||
+      ({
+        _class: 'color',
+        red: 0.5,
+        blue: 0.5,
+        green: 0.5,
+        alpha: 0.5,
+      } as Sketch.Color),
+    [fontColor],
+  );
+
+  const lineSpacing = useMemo(
+    () =>
+      selectedTextStyles.map(
+        (style) => style?.textStyle?.encodedAttributes.kerning,
```

Comment:
These 2 properties are flipped: `kerning` should be `letterSpacing` and `maximumLineHeight` should be `lineSpacing`